api/v1/design/resource.go

```diff
@@ -153,6 +153,64 @@ var _ = Service("resource", func() {
 		})
 	})
 
+	Method("ByCatalogKindNameVersionReadme", func() {
+		Description("Find resource README using name of catalog & name, kind and version of resource")
+		Payload(func() {
+			Attribute("catalog", String, "name of catalog", func() {
+				Example("catalog", "tektoncd")
+			})
+			Attribute("kind", String, "kind of resource", func() {
+				Enum("task", "pipeline")
+			})
+			Attribute("name", String, "name of resource", func() {
+				Example("name", "buildah")
+			})
+			Attribute("version", String, "version of resource", func() {
+				Example("version", "0.1")
+			})
+
+			Required("catalog", "kind", "name", "version")
+		})
+		Result(types.ResourceVersionReadme)
```

Comment:
I think we should return the response in terms of 
```
{
    data: {
       readme: "#Readme\n Sample readme content"
   }
}

Like we have for all resources api response 🙂 

And this would go same with the yaml as well